<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shezhi</class>
 <widget class="QWidget" name="shezhi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>背景音乐</string>
   </property>
   <widget class="QTextBrowser" name="music_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="music_settled">
       <property name="text">
        <string>选择默认音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="user_music">
       <property name="text">
        <string>选择自定义音乐</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="background_music_list">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>54</y>
      <width>329</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>329</width>
      <height>25</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Wake</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bgm1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Duvet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ephemeral_Memories</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Firefly_Glimmers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Komorebi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Make_Haste_Partern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perpetual_Motion_of_Wind</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Duel!_The_Summoners'_Summit!</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SecertSummerParadise</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>As_Light_Rain_Falls_Without_Reason</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>The_Exquisite_Night_Chimes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>An_Everlasting_Dream_Intertwined</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>棋盘大小</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>行                列</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_y">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="select_x">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>棋局设置</string>
   </property>
   <widget class="QRadioButton" name="jinshou_true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开启禁手</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="jinshou_false">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>不开启禁手</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LC">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>长连禁手</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FF">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>四四禁手</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TT">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>三三禁手</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>871</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/pictures/board_background.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>99</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>ai难度</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>入门</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>简单</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>中等</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>困难</string>
    </property>
   </widget>
  </widget>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>reset</zorder>
  <zorder>cancel</zorder>
  <zorder>confirm</zorder>
  <zorder>groupBox_4</zorder>
 </widget>
 <tabstops>
  <tabstop>music_settled</tabstop>
  <tabstop>background_music_list</tabstop>
  <tabstop>user_music</tabstop>
  <tabstop>music_name</tabstop>
  <tabstop>select_y</tabstop>
  <tabstop>select_x</tabstop>
  <tabstop>jinshou_true</tabstop>
  <tabstop>LC</tabstop>
  <tabstop>FF</tabstop>
  <tabstop>TT</tabstop>
  <tabstop>jinshou_false</tabstop>
 </tabstops>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>shezhi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
